topics/sequence-analysis/tutorials/ncbi-fcs/tutorial.md

```diff
@@ -14,14 +14,16 @@ key_points:
 - Contamination cleanup is essential for reliable comparative genomics analyses.
 contributors:
```

Comment:
in order to annotate funding you'll need to use the newer contributions (changed key) syntax:

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.